components/collections/TokenCard.tsx

```diff
@@ -61,6 +56,15 @@ export default ({
 
   const is1155 = token?.token?.kind === 'erc1155'
 
+  const price = token?.market?.floorAsk?.price?.amount?.raw
+    ? Number(
+        formatUnits(
```

Comment:
How many decimals does formatUnits return? I was thinking we could just pass the raw bigint straight into the FormatCryptoCurrency component